TACC/Core-Portal
# task/TV3-129 Update Request New Allocation Link

## Overview

Updated the Request New Allocation button to link to external TXRAS site

## Related

* [TV3-129](https://jira.tacc.utexas.edu/browse/TV3-129)

## Changes

- Replaced the previous Link component with an anchor tag that links to the external TXRAS site
- Added a test to ensure button is present and links to the correct site

## Testing

1. Go to the Allocations tab in the portal
2. Click on Request New Allocation and ensure external site opens 


## Notes
